test/UnitTestParquetCpp.chpl

```diff
@@ -122,9 +122,13 @@ proc testVersionInfo() {
   }
   var ret;
   try! ret = createStringWithNewBuffer(cVersionString);
-  if ret[0]: int >= 5 {
+  try {
+    // Ensure that version number can be cast to int
+    // Not checking version number for compatability
```

Comment:
is there a minimum version number for Aarow/Parquet?

Reply:
Not that I am aware of. Arrow version 4 is the oldest that we have tested with, so we could just require version 4, but I suspect it works with older versions as well. Note that the most recent version is `7.0.0`.